apis/infrastructure/v1beta1/byoclustertemplate_webhook.go

```diff
@@ -36,19 +35,12 @@ var _ webhook.Validator = &ByoClusterTemplate{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (byoClusterTemplate *ByoClusterTemplate) ValidateCreate() error {
 	byoclustertemplatelog.Info("validate create", "name", byoClusterTemplate.Name)
-	if errList := validateByoClusterSpec(byoClusterTemplate.Spec.Template.Spec); errList != nil {
```

Comment:
we don't need this validation anymore?

Reply:
yes I am using variables for bundleLookupTag while creating cluster, so it will be handled by ByoCluster webhooks in case it is empty.

Reply:
So this webhook is pretty much empty. Can we simply remove it?

Reply:
I meant remove the file only! Why do we even have to register a webhook if it is not being used?

Reply:
done